generate_documentation.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+#Must be called from the wanted output directory
+
+#Path to directory containing Dockerfile
+PATH_TO_IMAGE="$1"
+#Path to CGAL root directory
+PATH_TO_CGAL="$2"
+
+#build docker image
+docker build -t cgal_doc $PATH_TO_IMAGE
+#generate documentation
+docker run -v "$PATH_TO_CGAL":/mnt/cgal -t cgal_doc /docker_entrypoint.sh cgal_build_documentation
+#run second time to link
+docker run -v "$PATH_TO_CGAL":/mnt/cgal -t cgal_doc /docker_entrypoint.sh cgal_build_documentation
```

Comment:
Those runs of the container cannot work: between the two runs, the content of all the filesystem of the previous run will be lost! Containers are one shot and self-contained (but in Docker volumes).

If one needs to run twice the `make doc` step, then let's do in in the script ran in the container.